src/java/com/zimbra/oauth/handlers/impl/FacebookContactsImport.java

```diff
@@ -297,10 +297,7 @@ public void importData(List<Integer> folderIds, boolean fullSync) throws Service
                 // Facebook nextPageUrl if defined
                 final String url = buildContactsUrl(nextPageUrl, refreshToken);
                 nextPageUrl = null;
-                // log this only at the most verbose level, because this
-                // contains
-                // privileged information
-                ZimbraLog.extensions.debug("Attempting to sync Facebook contacts. URL: %s", url);
+                ZimbraLog.extensions.debug("Attempting to sync Facebook contacts.");
```

Comment:
logging without context will not be helpful when sync for multiple users will be in progress. Better to add the email id for which sync is happening if it's not too much overhead to get this information.

Reply:
As seen in the DEBUG log for this user (test@zcs-dev.test) there context of the Zimbra user as well as the DS name (Barbara.Seligsteinwitz-contact).

2018-07-03 21:35:32,207 DEBUG [ImportData-4] [name=test@zcs-dev.test;mid=2;ds=Barbara.Seligsteinwitz-contact;] extensions - Attempting to sync Facebook contacts.

Reply:
2018-07-03 22:32:37,620 INFO  [ImportData-1] [name=test@zcs-dev.test;mid=2;ds=Barbara.Seligsteinwitz-contact;] extensions - Attempting to sync Facebook contacts.